examples/Raytracer_Benchmark/Color.h

```diff
@@ -0,0 +1,194 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+/// \file Color.h
+/// \author Andrei Gheata
+
+#ifndef VECGEOM_BENCHMARKING_COLOR_H_
+#define VECGEOM_BENCHMARKING_COLOR_H_
+
+#include "base/inc/CopCore/include/CopCore/Global.h"
```

Comment:
Style issue: preferably use <> instead of double quotes for including non-local files